docs/index_white.nim

```diff
@@ -6,28 +6,28 @@ initReveal() # init the slide and introduce templates and variables
 setSlidesTheme(White)
```

Comment:
are index and index_white the same except for this line? if so you could use either a CT or RT option and avoid duplicating code

Reply:
Smart! Will do so instead :D